flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/JsonFormatOptions.java

```diff
@@ -73,6 +73,13 @@
                     .withDescription(
                             "Optional flag to specify whether to encode all decimals as plain numbers instead of possible scientific notations, false by default.");
 
+    public static final ConfigOption<Boolean> ALLOW_NON_NUMERIC_NUMBERS =
+            ConfigOptions.key("allow-non-numeric-numbers")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription(
+                            "Optional flag to specify whether to deserialize non-numeric number instead of failing, false by default.");
```

Comment:
Can we remove "optional" and "false by default" here? Both of those information are defined elsewhere (`defaultValue` and `optionalOptions`) and thus create duplication.

Reply:
Thanks for your review!  I'm concerned about that it will break the same writing style with above options.

Reply:
The above options should just be changed as well. Mentioning the default in the description is OK if some explanation is needed. Otherwise it's just duplicating the information in the documentation.